setup.md

```diff
@@ -0,0 +1,72 @@
+# Setting up your local development environment for PyGMT
+
+We will use Git, Anaconda, and the `conda` package manager to install and
+manage the development environment.
+
+This guide assumes a basic understanding of git using the command line. For
+more help, attend the optional setup session on August 17 and/or review the
+[Software Carpentry](https://software-carpentry.org/)'s courses on
+[the Unix Shell](https://swcarpentry.github.io/shell-novice/) and
+[Version Control with Git](https://swcarpentry.github.io/git-novice/).
+
+## Cloning and forking the repository
+
+**Note**: The steps in this section only need to be performed once per
+repository.
+
+1. **Clone the repository.**
+
+   - In a browser, navigate to the
+     [PyGMT repository](https://github.com/GenericMappingTools/pygmt).
+   - Login to GitHub by clicking the 'Sign In' button in the top-left corner.
+   - Click the green 'Code' button ①.
+   - If you do not have
+     [ssh keys set up](https://docs.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh),
+     use the https url by first clicking the `https` button ②.
+   - Copy the link by clicking the clipboard icon ③.
+   - In a terminal window, navigate to the directory where you want to place
+     the cloned PyGMT repository using `cd`.
+   - Type `git clone clone-url` into the terminal (replace `clone-url`
+     with the url that was copied to your clipboard).
+
+   ![clone.png](images/clone.png)
```

Comment:
In case you don't know it, the [Squoosh.app](https://squoosh.app/) site is a useful online tool to compress images. The 400 KB clon.png image can be losslessly compressed to less than 100 KB using the default parameters.

The image sizes don't matter much for this repository. So it's up to you to decide if you want to compress the images to reduce the repo size.

Reply:
Thanks, this looks like a great tool! I'm not going to bother replacing these two images but will compress any future images for the repository before adding.